### PR TITLE
Add Monolith Architecture

### PR DESCRIPTION
**Pull request by canerbasaran**
**[https://github.com/canerbasaran/awesome-monolith#readme]**

**[List of content describing the benefits of starting projects with a monolith architecture]**

### CODE REVIEW

**Comment by jluterek** [comment]
This reads as anti-microservices. Is that something to build a community around and more importantly... is it awesome?

**Comment by canerbasaran** [comment]
> This reads as anti-microservices. Is that something to build a community around and more importantly... is it awesome?

According to this discourse, I'd say microservices are anti-monolith. There are many unsuccessful projects started with Microservices. Microservices aren't a silver bullet. You should choose the architecture according to the need. Nothing gives a magic solution.

File: readme.md [hunk 1]
```diff
@@ -896,6 +896,7 @@
 - [Computational Geometry](https://github.com/atkirtland/awesome-computational-geometry#readme) - Computational approaches for problems in geometry.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
```
Comment (mic4aelx504) [inline readme.md:898]: E

**Review by mic4aelx504 — APPROVED** [review]

**Review by mic4aelx504 — APPROVED** [review]

**Review by mic4aelx504 — APPROVED** [review]

**Review by mic4aelx504 — CHANGES_REQUESTED** [review]
Go

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Review by andrzejsydor — APPROVED** [review]

**Comment by vhladiienko** [comment]
Hi @canerbasaran 

I don't think the linter will pass. You can run it yourself locally by executing:

`npx awesome-lint https://github.com/canerbasaran/awesome-monolith`

But make sure you do it at least twice, as it currently stops linting at the `remark-lint:awesome-toc` issue, whereas I can see some other stuff that should not pass, e.g. wrong list entry formatting (you should have dashes between a link and a description).

**Comment by canerbasaran** [comment]
Thanks @vhladiienko for suggestions

**Comment by sindresorhus** [comment]
You are not following the formatting expected for Awesome lists.

**Comment by sindresorhus** [comment]
I think in general the list needs more work. It feels a bit unfinished and not up to the quality expected from Awesome lists.

File: readme.md [hunk 2]
```diff
@@ -900,6 +900,7 @@
...
+- [Monolith Architecture](https://github.com/canerbasaran/awesome-monolith#readme) - Traditional model of a software program.
```
Comment (sindresorhus) [inline readme.md:903]: The description is not very useful. It doesn't actually say what a monolith is.

**Comment by pee563** [comment]
> You are not following the formatting expected for Awesome lists.

So someone is in my account controlling aspects of my phone. I have an idea who it is. It's is causing me not to be able to feed my family because he is shutting work apps.

**Comment by dontbethatguy209** [comment]
Please advise, many attempts to figure out objective but still not getting it. What can yoh do to help?

Sent via the Samsung Galaxy S21 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by andrzejsydor — APPROVED** [review]

**Comment by Simeon221** [comment]
091eefb4d3b2903fb15d94316e0c64ce9072db42

**Pull request closed without merging**